cmd/execution/main.go

```diff
@@ -318,6 +334,71 @@ func main() {
 
 			return compactor, nil
 		}).
+		Component("execution data service", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			err := os.MkdirAll(executionDataDir, 0700)
+
+			if err != nil {
+				return nil, err
+			}
+
+			ds, err := badger.NewDatastore(executionDataDir, &badger.DefaultOptions)
```

Comment:
we need to add some logic to close the datastore after the execution data service has stopped. 

One idea I had was to add a chainable method to scaffold that lets you register a function to be called when `Node` calls the `postShutdown` callback.

Reply:
23d4538

Reply:
I put a PR together that implements this: https://github.com/onflow/flow-go/pull/1907